README.md

```diff
@@ -40,6 +40,14 @@ To run the test-suite locally, first complete the bootstrap step as outline abov
 tox -e tests
 ```
 
+### Documentation
+
+To generate html documentation locally in docs/_build/html, run the following command
```

Comment:
s/html/HTML/
I would put "docs/_build/html" between backtick => `docs/_build/html` (otherwise you need to escape the `_`).

Reply:
done